packages/block/src/types.ts

```diff
@@ -89,7 +89,7 @@ export interface HeaderData {
   stateRoot?: BufferLike
   transactionsTrie?: BufferLike
   receiptTrie?: BufferLike
-  bloom?: BufferLike
+  logsBloom?: BufferLike
```

Comment:
What is the main motivation for this change (not against it, just asking)?

Reply:
long overdue, part of my breaking changes plan from a few months ago, but I thought since I can add backward compatibility to ensure no downstream usage breaks, this is a good opportunity to align our keys with the expected `ExecutionPayload` fields.